packages/backend-common/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@backstage/backend-common",
   "description": "Common functionality library for Backstage backends",
-  "version": "0.5.0",
```

Comment:
You don't need to bump the version yourself. Adding the changeset file is enough. :)